force-app/test/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": "eslint:recommended",
+    "plugins": ["jasmine"],
+    "env": {
+        "node": true,
+        "es2021": true,
+        "jasmine": true
+    },
+    "globals": {
+        "utam": "readonly",
+        "browser": "readonly"
```

Comment:
I think we can delete the `browser` global and use [`eslint-plugin-wdio`](https://www.npmjs.com/package/eslint-plugin-wdio) instead. Note that it should match the version range of the other WDIO dependencies.